fe/fe-core/src/main/java/org/apache/doris/catalog/TableIf.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.doris.common.DdlException;
 import org.apache.doris.common.MetaNotFoundException;
 import org.apache.doris.nereids.exceptions.AnalysisException;
```

Comment:
maybe we should not throw nereids exception in catalog class

Reply:
Yeah, we can log error rather than throw exception.